Jenkinsfile_main_develop

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/groovy
+
+@Library('cbci_shared_libs@develop') _
```

Comment:
Keep it as develop for now, once we are getting used to it remove the @develop branch tag.